src/components/Task/MassActions/ConfirmationModal/MassActionsTasksConfirmationModal.tsx

```diff
@@ -20,23 +27,71 @@ export const MassActionsTasksConfirmationModal: React.FC<
 > = ({ open, idsCount, action, setOpen, onConfirm }) => {
   const { t } = useTranslation();
 
+  // KG: useState for checkbox
```

Comment:
```suggestion
```

Reply:
The way we write our code, we generally only write comments when needed. Instead, we make variable and function names very descriptive to allow the user to read the code without needing comments.

However, there are cases were comments are needed